pom.xml

```diff
@@ -52,13 +52,26 @@
 			<artifactId>spring-boot-starter-test</artifactId>
 			<scope>test</scope>
 		</dependency>
+			<dependency>
+				<groupId>org.projectlombok</groupId>
+				<artifactId>lombok</artifactId>
+				<version>1.18.16</version>
+				<scope>provided</scope>
```

Comment:
Je to spravne, zase jen poznamka, ze je treba si ucedomit, ze lombok se vyhodnocuje v dobe kompilace a nebere tedy zadny procesorovy cas v dobe behu. To znamena, ze scope by nemel byt napriklad compile, protoze si neprejeme, aby ve vyslednem JARku byla knihovna, ktera se realne nepouziva. Proto je pouziti  "provided" na spravnem miste.